.github/ISSUE_TEMPLATE/new-package.md

```diff
@@ -0,0 +1,28 @@
+---
+name: New package
+about: Suggest software for inclusion in the repo
+title: Package <name>
+labels: package
```

Comment:
Should we add a label for requests?

Reply:
I don't think that's necessary, we would just have
- Issues labeled with `package`: package requests
- PRs labeled with `package`: added package

Reply:
ok.